pkg/pmem-csi-operator/main.go

```diff
@@ -16,13 +16,15 @@ import (
 	"k8s.io/klog/v2"
 
 	"github.com/intel/pmem-csi/pkg/apis"
-	api "github.com/intel/pmem-csi/pkg/apis/pmemcsi/v1beta1"
+	"github.com/intel/pmem-csi/pkg/apis/pmemcsi/v1beta1"
 	"github.com/intel/pmem-csi/pkg/k8sutil"
 	"github.com/intel/pmem-csi/pkg/pmem-csi-operator/controller"
 
 	pmemcommon "github.com/intel/pmem-csi/pkg/pmem-common"
-
 	"github.com/operator-framework/operator-lib/leader"
```

Comment:
nit: I find it useful to separate "our" imports from Go runtime imports (at the top) and third-party imports (below).

In other words, don't delete this blank line.

Reply:
Or rather, delete line 22 above this.

Reply:
Fixed.